src/api/test/functional/status_controller_test.rb

```diff
@@ -73,14 +73,33 @@ def test_bsrequest
   def test_history
     get "/status/history"
     assert_response 400
-
-    get "/status/history?hours=24&key=idle_i586"
+    assert_select "status", code: "missing_parameter" do
+      assert_select "summary", "Required Parameter hours missing"
+    end
+
+    Timecop.freeze(2010, 7, 12)
+    day_before_yesterday = Time.now.to_i - 2.days
+    yesterday = Time.now.to_i - 1.days
+    1.times do |i|
+      StatusHistory.create(time: day_before_yesterday + i, key: 'squeue_low_aarch64', value: i)
+    end
+    2.times do |i|
+      StatusHistory.create(time: yesterday + i, key: 'squeue_low_aarch64', value: i)
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.